plugin-code/src/main/java/com/trivago/cluecumber/properties/PropertyManager.java

```diff
@@ -29,6 +29,7 @@
 
     private final CluecumberLogger logger;
 
+    private String skip;
```

Comment:
In my opinion, the skip flag is not in line with the other properties (since its purpose is just to skip the plugin execution directly) so it doesn't need to be stored or logged in the PropertyManager.